docs/usage_general.rst.inc

```diff
@@ -60,10 +60,28 @@ Repository / Archive Locations
 Many commands want either a repository (just give the repo URL, see above) or
 an archive location, which is a repo URL followed by ``::archive_name``.
 
-Archive names must not contain the ``/`` (slash) character. For simplicity,
-maybe also avoid blanks or other characters that have special meaning on the
-shell or in a filesystem (borg mount will use the archive name as directory
-name).
+Borg does not assume a particular meaning or structuring of archive names,
+but for use with :ref:`borg_prune` and other commands it makes sense
+to structure names with prefixes (e.g. ``{hostname}-{date}``) for the
+``--prefix`` option.
+
+When mounting an entire repository with :ref:`borg_mount` (without the
+'versions' option) the archive names are used as paths in the
+virtual filesystem.
```

Comment:
well, this somehow contradicts the first paragraph.
after this proposed change, borg mount assumes that the archive name is a path (without a slash it is still a 1-level path).